src/app/parcels/parcelsTable/ParcelsTable.tsx

```diff
@@ -226,15 +225,22 @@ const ParcelsTable: React.FC<ParcelsTableProps> = ({
         }
     }, [areFiltersLoadingForFirstTime, fetchAndDisplayParcelsData]);
 
-    const packingManagerViewDataPortion = (
-        parcelsDataPortion: ParcelsTableRow[]
-    ): ParcelsTableRow[] => {
-        return parcelsDataPortion.filter((parcel) => {
-            if (shouldBeInPackingManagerView(parcel, today, yesterday)) {
-                return parcel;
-            }
-        });
-    };
+    const packingManagerViewDataPortion = React.useCallback(
```

Comment:
Very minor point: we're already importing `useCallback`, so we don't have to do `React.` and can avoid importing `React`